src/index.tsx

```diff
@@ -2,7 +2,9 @@ import React from 'react';
 import { createRoot } from 'react-dom/client';
 import 'wicg-inert';
 import { registerSW } from 'virtual:pwa-register';
+import 'reflect-metadata'; // include once in the app for inversify (see: https://github.com/inversify/InversifyJS/blob/master/README.md#-installation)
 
+import '#src/modules/init2';
```

Comment:
imho, it's more obvious what's happening to have the init be a method call in the root and it also gives easier access to queries and such in the application context where we have all the components and context to use them. Maybe there's some unification that can happen between index, App, and Root though?

Reply:
We will have an explicit `applicationController.initializeApp` call + general import here to register services / controller.